docs/source/guides/logical_types_and_semantic_tags.ipynb

```diff
@@ -78,9 +78,10 @@
     "    - A time index column will contain either datetime or numeric data\n",
     "\n",
     "#### Other Tags\n",
-    "This section lists any tags that Woodwork will neither add to a DataFrame nor take direct action upon, but that are recognized to have a special meaning.\n",
+    "The tags listed below may be added directly to columns during or after Woodwork initialization. They are tags that have suggested meanings and that can be added to columns that will be used in the manner described below. Woodwork will neither add them automatically to a DataFrame nor take direct action upon a column if they are present. \n",
```

Comment:
@thehomebrewnerd is this along the lines of what you were thinking for this section?

Reply:
Yeah, I like that a lot better given the discussion about these tags.

Reply:
Awesome, thanks for the direction here!